### PR TITLE
Modifying operator_converters for keras to improve onnx node naming

### PR DESCRIPTION
**Pull request by Oewyn**
I found it challenging to do any sort of comparison between an onnx converted graph and the original keras graph because almost all of the names didn't match up. This PR is an attempt to address that. I have not modified all the keras operator_converters, but I did hit most of them (the ones remaining are namely RNN related -- Bidirectional, Embed, GRU, LSTM, SimpleRNN)

Any feedback would be helpful, I believe i got all the scope.get_unique_operator_name(...) vs scope.get_unique_variable_name(...) separated correctly.

### CODE REVIEW

**Comment by Oewyn** [comment]
Yes, I've made sure that this change doesn't introduce any illegal characters.  Everything is run through scope.get_unique_* which checks and modifies keras supplied names and replaces illegal characters with '_'

File: onnxmltools/convert/keras/operator_converters/AdvancedActivation.py [hunk 1]
```diff
@@ -11,20 +11,21 @@
 
 def convert_keras_advanced_activation(scope, operator, container):
     op = operator.raw_operator
+    keras_name = scope.get_unique_operator_name(operator.raw_operator.name)
     if isinstance(op, activations.LeakyReLU):
         alpha = op.get_config()['alpha']
         apply_leaky_relu(scope, operator.input_full_names[0], operator.output_full_names[0], container,
-                         operator_name=operator.full_name, alpha=alpha)
+                         operator_name=keras_name, alpha=alpha)
```
Comment (wenbingl) [inline onnxmltools/convert/keras/operator_converters/AdvancedActivation.py]: I didn't find the place where operator_name was referenced. Did I miss anything?

**Comment by wenbingl** [comment]
Please add an unit test to verify these name changing really works.

File: onnxmltools/convert/keras/operator_converters/AdvancedActivation.py (hunk 1 above)
Comment (Oewyn) [reply onnxmltools/convert/keras/operator_converters/AdvancedActivation.py]: operator_name is a named parameter to the apply_leaky_relu function. All apply_* functions have this parameter that allows the caller to set the name of the node container that is created.

**Comment by Oewyn** [comment]
Thanks for the feedback, I should be able to get to the unit_tests early next week.

**Comment by Oewyn** [comment]
Closing for now

**Pull request closed without merging**